### ISSUE TITLE
Calling Meteor.logout() results an error

### ISSUE THREAD

**Issue by NickTheTurtle** [TheRealNate/meteor-react-native#21]
**Describe the bug**
Calling Meteor.logout results an error.

**To Reproduce**
Call Meteor.logout() in project. User is logged out though. See screenshot below.

**Expected behavior**
User is logged out. No error.

**Screenshots**
The error below shows that this.handleLogout is not defined.
<img width="319" alt="image" src="https://user-images.githubusercontent.com/12247846/88248064-7e3e5980-cc65-11ea-95a5-eb68de26378a.png">

**Device (please complete the following information):**
 - Device: [e.g. iPhone11]
 - OS: [e.g. iOS13.6]
 - React Native and MeteorRN versions: [e.g. 0.63.2, 2.0.8]

**Comment by TheRealNate** [comment]
Hi @NickTheTurtle. Thanks for posting! I can't seem to reproduce the issue. I am testing on the same React Native and MeteorRN versions as you.

Any chance you could share the full error message and stacktrace?

If you have a chance, these further measures may help diagnose the issue:
- Try clearing your build folder in XCode then rebuilding
- Try uninstalling MeteorRN and reinstalling
- Try using a different version of iOS (e.g. 13.4)

**Comment by NickTheTurtle** [comment]
Thanks for the quick reply! Here's the more detailed error message. I'll try reinstalling MeteorRN and clearing the build folder.

<img width="342" alt="image" src="https://user-images.githubusercontent.com/12247846/88351375-b011f780-cd1b-11ea-9339-c1c122eca251.png">

**Comment by TheRealNate** [comment]
Hi @NickTheTurtle, any updates on this?

**Comment by NickTheTurtle** [comment]
It still doesn't work.

**Comment by TheRealNate** [comment]
Hmm, it seems like `this` is evaluating to the `global` object. This is really strange behavior (could also be related to your other issue #22). 

Do you have anything that could be changing how `this` is being bound?

**Comment by TheRealNate** [comment]
I just did some testing in the console. If you'd like, you can try changing the syntax:

From:
````
logout() {

}
````

To:
````
logout:function () {

}
````

**Comment by NickTheTurtle** [comment]
You're right. I had `<Button onPress={Meteor.logout}>Logout</Button>`, but React bound `this` to the global object instead of Meteor. I changed it to `<Button onPress={Meteor.logout.bind(Meteor)}>Logout</Button>` and now everything works perfectly.

**Comment by TheRealNate** [comment]
Hi @NickTheTurtle, thanks for providing your solution! I'm going to see if there is a way to fix this. If its not too much of an inconvenience, could you try something like this (which is how I have it in my apps), where Meteor logout is called from inside a function?:
````
// For class Component (<Button onPress={this.logout}>)
logout = () => {
    Meteor.logout();
}

// For function components (<Button onPress={logout}>)
const logout = () => {
    Meteor.logout()
}
````

**Comment by TheRealNate** [comment]
Hey @NickTheTurtle, one last thing. I've just published `2.0.10-beta1` which converts it from an object to a class. If its not too much of an inconvenience, could you try installing that version then using it the original way you were (`<Button onPress={Meteor.logout}>`)
